### DISCUSSION TITLE
Logs livestream

### DISCUSSION THREAD

**Discussion by martadinata666** [openobserve/openobserve#929] [General]
My use case is docker daemon log to openobserve via fluentd. The issue is the log table is keep reloading every 5s `the shortest` time I can get. Can we get a seamless live log? instead table keep reloading, let the text loading realtime and keep scrolling. This may be feature enhancement but I let maintainers decide.

Thanks.

**Comment by prabhatsharma** [comment]
This would be a feature and will require a decent amount of work. Can you please create an issue for it. This will help us prioritize it.
